framework/src/problems/FEProblemBase.C

```diff
@@ -292,6 +292,10 @@ FEProblemBase::validParams()
       "allow_invalid_solution",
       false,
       "Set to true to allow convergence even though the solution has been marked as 'invalid'");
+  params.addParam<bool>(
+      "show_invalid_solution_console",
+      false,
+      "Set to true to allow convergence but still show the invalid solution occurance in console");
```

Comment:
this overlaps with the function of allow_invalid_solution. I dont see why we would want to not show the table? Especially by default

Reply:
It will break the previous test cases who silences the warnings. Also if they have a long list, people may not want to show it in the console, similar to show_linear_iteration parameters

Reply:
this is hiding warnings now
we cant default to false

Reply:
the previous test case which silences it is allowing invalid but not showing the table? This doesnt seem right

Reply:
Yes, but it will give warning message
` mooseWarning("The Solution Invalidity warnings are detected but silenced! "
                     "Use Problem/show_invalid_solution_console=true to show solution counts");`

Reply:
let's just still default to true
those tests can change it to false

Reply:
I think it's fine since users still get .json output in the end when they have invalid solutions. Will that be enough for a reminder?

Reply:
I dont think so